### ISSUE TITLE
Compatible issue with dotnet v3.0

### ISSUE THREAD

**Issue by masq6r** [Zaid-Ajaj/Femto#59]
Installed by issuing `dotnet tool install femto --global` and run `femto` but the error pops up:
```
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '2.2.0' was not found.
  - The following frameworks were found:
      3.0.0 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]
```

Env:
```
CentOS 7 x64
dotnet core 3.0.100
fable-compiler 2.4.6
node.js v10
```

**Comment by Zaid-Ajaj** [comment]
Hello there @masq6r, 
This looks like an issue where you are trying to restore a netcoreapp2.2 version but it isn't working (regardless of Fable/node.js versions). Can you please check whether `dotnet restore` works by itself first? Thank you!

**Comment by masq6r** [comment]
> Hello there @masq6r,
> This looks like an issue where you are trying to restore a netcoreapp2.2 version but it isn't working (regardless of Fable/node.js versions). Can you please check whether `dotnet restore` works by itself first? Thank you!

Unfortunately it's not working. To reproduce the issue:

1. Clone fable2-samples.
2. Go to `browser` and `npm install`.
3. Go to `browser/src' and `dotnet restore`.
3. Issue `femto` command (`femto` was just installed for the user prior to this). You'll see the error message.

I have only one `dotnet` SDK installed which is 3.0.100.

**Comment by Zaid-Ajaj** [comment]
Hmm I am not able to reproduce the issue: 

![2019_10_28_16_40_35_Administrator_Developer_Command_Prompt_for_VS_2017](https://user-images.githubusercontent.com/13316248/67693397-e6dffa80-f9a1-11e9-9d7f-3130917977a1.png)

Maybe I need to publish a new version that targets netcoreapp3.0?

**Comment by Zaid-Ajaj** [comment]
> Running femto --version also showed me the same as above. dotnet tool list -g showed me femto is v0.8.0 rather than v0.7.0 like yours. And suggestions or more information to troubleshoot this? Thanks.

This is just me forgetting to match the version number in the code (what you see in the terminal) with the version of the published nuget. I just fixed that now and published a new version as of 0.8.1 with target framework netcoreapp3.0, can you take you for a spin to see if it resolves your issue?

**Comment by masq6r** [comment]
> > Running femto --version also showed me the same as above. dotnet tool list -g showed me femto is v0.8.0 rather than v0.7.0 like yours. And suggestions or more information to troubleshoot this? Thanks.
> 
> This is just me forgetting to match the version number in the code (what you see in the terminal) with the version of the published nuget. I just fixed that now and published a new version as of 0.8.1 with target framework netcoreapp3.0, can you take you for a spin to see if it resolves your issue?

Thank you very much. I confirm that the fix is valid.
